addons/project/models/project.py

```diff
@@ -333,6 +333,8 @@ def _compute_partner_phone(self):
         for project in self:
             if project.partner_id and project.partner_phone != project.partner_id.phone:
                 project.partner_phone = project.partner_id.phone
+            elif not project.partner_id:
+                project.partner_phone = ''
```

Comment:
Why don't we do the same for the `partner_email` field? 🙂

Reply:
Moreover, I'm not sure but when we have no `partner_id` then the `task.partner_id.phone` is equal to an empty string, isn't it? 🤔 
If it is the then we could just do this:
 ```py
    @api.depends('partner_id.phone')
    def _compute_partner_phone(self):
        for project in self:
            if project.partner_phone != project.partner_id.phone:
                project.partner_phone = project.partner_id.phone
```
I am really not sure for this. ^^
What do you think? 🙂

Reply:
It worked, I changed it for both the email and the phone :+1: